README.md

```diff
@@ -42,47 +42,41 @@ devtools::install_github("2DegreesInvesting/r2dii.climate.stress.test")
 
 ## Example
 
-  - Use `library()` to attach the package
-
-<!-- end list -->
+-   Use `library()` to attach the package
 
 ``` r
 library(r2dii.climate.stress.test)
 ```
 
-  - Stress tests for corporate loans require a preparatory step for
+-   Stress tests for corporate loans require a preparatory step for
     input data preparation for initial application of stress testing on
     a loan book
 
-<!-- end list -->
-
 ``` r
 run_prep_calculation_loans(
-  input_path_project_specific = "/path/to/specific/data",
-  input_path_project_agnostic = "/path/to/agnostic/data",
-  data_prep_output_path = "/path/to/specific/data"
+  input_path_project_specific = "/example_project/project_specific_input/",
+  input_path_project_agnostic = "/example_project/project_agnostic_input/",
+  output_path = "/example_project/output",
```

Comment:
should this be input instead of output ? according to the vignette https://2degreesinvesting.github.io/r2dii.climate.stress.test/articles/02-prepare-loans-inputs.html

Reply:
This PR merely renders the content of readme.Rmd, I did not do any manual changes. But thanks for pointing me to this, the argument name data_prep_output_path was incorrectly renamed to output_path in https://github.com/2DegreesInvesting/r2dii.climate.stress.test/commit/4d166a9daa56b00b103a789d2863ae092cb962bb (fyi @jakubcervenka ) I will fix it.